code/game/objects/structures/window.dm

```diff
@@ -69,6 +69,14 @@
 
 	air_update_turf(TRUE)
 
+/obj/structure/window/proc/toggle_polarization()
```

Comment:
How does this interact with window painters? Does the colour get overwritten?

Reply:
You cannot paint electrochromic windows, IIRC.

Reply:
What about colourful reagent / paint from xenobio?

Reply:
Added a method to remember old colours, resolved.